docs/app/Examples/modules/Dropdown/Types/Dropdown.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import { Dropdown } from 'stardust'
+
+export default class DropdownDropdownExample extends Component {
+  render() {
+    return (
+      <Dropdown text='File' action='hide'>
```

Comment:
If you only want to show a menu, and not have the text update, set the correct [Semantic UI action](http://semantic-ui.com/modules/dropdown.html#specifying-select-action).  In this case, we just hide the dropdown on select.  This leaves the `File` text in the dropdown space so it looks and works like a normal dropdown menu (not a selection dropdown).